src/js/jwt/jwt.js

```diff
@@ -98,6 +99,8 @@ exports.init = (options) => {
     options.url = insightsUrl(((options.routes) ? options.routes : DEFAULT_ROUTES));
     options.promiseType = 'native';
 
+    options.timeSkew = 10;
```

Comment:
Why are we setting this for every user?

Reply:
This is just a default. If Keycloak can't determine the actual time skew, it will use this value. Currently, when I run this, the skew is actually 0. There are many functions that we aren't using in here (yet?) that require a default time skew.